### PR TITLE
Add tests for otc trades

### LINKED ISSUE #292: Write OTC Trade UI Tests
## Problem Definition

We need UI tests for adding, removing and editing an external trade.

## Task

- [ ] Write a test that adds two external trade via the OTC Trades pane and sees them in the OTC Trades table
- [ ] Then write a test that edits one trade and sees the change in the table
- [ ] Finally write a test that removes one trade and sees it removed from the table.

### PR DESCRIPTION
**Pull request by kelsos**
Closes #292

### CODE REVIEW

**Comment by kelsos** [comment]
macOS tests failed due to 
https://i.imgur.com/UnOyl1P.png

**Comment by LefterisJP** [comment]
@kelsos they seem to have passed now? You restarted them? The error you saw there seemed only on the binance side.

**Comment by kelsos** [comment]
Yes, I restarted them and kept the link to the screenshot just for documenting the failure.

File: main.js [hunk 1]
```diff
@@ -30,6 +30,28 @@ function disableAnimationsForTest() {
     }
 }
 
+function setupInspectMenu() {
```
Comment (LefterisJP) [inline main.js]: @kelsos What is this function for?

Comment (kelsos) [reply main.js]: This adds the `inspect` action on the context menu for development, but without having to depend on third-party libraries. I already checked that this is not enabled for the packaged build, but I would like if you could also verify it just to be on the safe side.

Comment (LefterisJP) [reply main.js]: But I just checked and this seems to be always on. Can you make it activated only by a switch or something? People can use rotkehlchen from source and does not mean they want to have development tools on by default.

Comment (kelsos) [reply main.js]: You mean by using `npm start`, yes this was intentional. I was referring to the packaged build. I can change it, though I cannot imagine why would anyone go through the trouble of installing node.js and all the dependencies other than coding :P

Comment (LefterisJP) [reply main.js]: There are people like me, Rakan and probably you who use the program by source. So the end result should by default be like what the final binary should be.

Comment (LefterisJP) [reply main.js]: Just make this a switch we can activate when developing to enable the right click context menu. @kelsos ... do you need this in the automatic UI tests?

Comment (kelsos) [reply main.js]: I need it to build the tests mostly

**Comment by LefterisJP** [comment]
@kelsos Thanks  lot man. Really awesome to have tests for OTC trades.

**Pull request merged**